python/cuml/decomposition/pca.pyx

```diff
@@ -525,6 +629,9 @@ class PCA(Base):
             Acceptable formats: cuDF DataFrame, NumPy ndarray, Numba device
             ndarray, cuda array interface compliant array like CuPy
 
+            sparse array-like (device) shape = (n_samples, n_features)
+            Acceptable formats: cupy.sparse
```

Comment:
We should probably accept scipy.sparse here as well and put it on device to be consistent with other sparse estimators.